server/schemas/item.graphql

```diff
@@ -0,0 +1,74 @@
+type Item implements Node {
+  id: ID!
+  modelId: ID!
+  model: Model!
+  values: [ItemValue!]!
+  publicVersion: String!
+  #  draftVersion: String!
+  createdAt: DateTime!
+  updatedAt: DateTime!
+}
+
+type ItemValue {
+  fieldId: ID!
+  #  version: String!
+  value: Any!
+}
+
+# Inputs
+input ItemValueInput {
+  fieldId: ID!
+  value: Any!
+}
+
+input CreateItemInput {
+  modelId: ID!
+  values: [ItemValueInput!]!
+}
+
+input UpdateItemInput {
+  itemId: ID!
+  values: [ItemValueInput!]!
+}
+
+input DeleteItemInput {
+  itemId: ID!
+}
+
+# Payloads
+type ItemPayload {
+  item: Item!
+}
+
+type DeleteItemPayload {
+  itemId: ID!
+}
+
+type ItemConnection {
+  edges: [ItemEdge!]!
+  nodes: [Item]!
+  pageInfo: PageInfo!
+  totalCount: Int!
+}
+
+type ItemEdge {
+  cursor: Cursor!
+  node: Item
+}
+
+extend type Query {
+  items(
+    projectId: ID!
```

Comment:
```suggestion
    modelId: ID!
```

Remember relation between domain models: `Project *- Model - Schema *- Item`. Also note that this query returns latest versions of each item.